using-mobile-extensions/adobe-target/README.md

```diff
@@ -238,6 +369,156 @@ NSDictionary *profileParameters = @{@"age":@"20-32"};
++ (nonnull instancetype) targetOrderWithId: (nonnull NSString*) orderId
+                                     total: (nullable NSNumber*) total
+                       purchasedProductIds: (nullable NSArray <NSString*>*) purchasedProductIds;
+
++ (nonnull instancetype) targetProductWithId: (nonnull NSString*) productId
+                                  categoryId: (nullable NSString*) categoryId;
+
++ (nonnull instancetype) targetParametersWithParameters: (nullable NSDictionary*) parameters
+                                      profileParameters: (nullable NSDictionary*) profileParameters
+                                                product: (nullable ACPTargetProduct*) product
+                                                  order: (nullable ACPTargetOrder*) order;
+```
+
+#### Objective-C Example
+
+```objectivec
+NSDictionary *mboxParameters = @{@"status":@"Platinum"};
+NSDictionary *profileParameters = @{@"gender":@"female"};
+
+ACPTargetProduct *product = [ACPTargetProduct targetProductWithId:@"24D334" categoryId:@"Stationary"];
+
+ACPTargetOrder *order = [ACPTargetOrder targetOrderWithId:@"ADCKKBC" total:@(400.50) purchasedProductIds:@[@"34", @"125"]];
+
+ACPTargetParameters *targetParameters = [ACPTargetParameters targetParametersWithParameters:mboxParameters
+                                                    profileParameters:profileParameters
+                                                              product:product
+                                                                order:order];
+```
+{% endtab %}
+{% endtabs %}
+
+### Using Locations Displayed API for Prefetch
+
+In some situations, you may want to want to inform Target that the corresponding location (mbox) has been viewed. This API sends a display notification to Target for a given prefetched mbox. This helps Target record location display events.
+
+Note: If you're only using regular mboxes, and not prefetching any mbox content, this method should not be called.
+
+{% tabs %}
+{% tab title="Android" %}
+#### **Syntax**
+
+```java
+public static void locationsDisplayed(final List<String> mboxNames, final TargetParameters parameters)
+```
+
+#### **Example**
+
+```java
+List<String> purchasedProductIds = new ArrayList<String>();
+purchasedProductIds.add("34");
+purchasedProductIds.add("125"); 
+TargetOrder targetOrder = new TargetOrder("123", 567.89, purchasedProductIds);
+
+TargetProduct targetProduct = new TargetProduct("123", "Books");
+TargetParameters targetParameters = new TargetParameters.Builder()
+                                    .parameters(new HashMap<String, String>())
+                                    .profileParameters(new HashMap<String, String>())
+                                    .product(targetProduct)
+                                    .order(targetOrder)
+                                    .build();
+List<String> mboxList = new ArrayList<>();
+mboxList.add("mboxName1");
+Target.locationsDisplayed(null, targetParameters);
```

Comment:
Why are we passing null instead of `mboxList`  here?

Reply:
missed it. Added `mboxList`